src/systems/filecoin_mining/storage_mining/storage_miner_actor_state.go

```diff
@@ -65,18 +67,18 @@ func (st *StorageMinerActorState_I) _updateSectorUtilization(rt Runtime, lastPoS
 
 }
 
-func (st *StorageMinerActorState_I) _getActivePower(rt Runtime) block.StoragePower {
+func (st *StorageMinerActorState_I) _getActivePower() (block.StoragePower, error) {
 	var activePower = block.StoragePower(0)
 
 	for _, sectorNo := range st.SectorTable().Impl().ActiveSectors_.SectorsOn() {
-		utilizationInfo, found := st.SectorUtilization()[sectorNo]
-		if !found {
-			rt.AbortStateMsg("sm._getActivePower: sectorNo not found in SectorUtilization")
+		utilInfo, err := st._getUtilizationInfo(sectorNo)
```

Comment:
Blocking: Any reason why we are not using safeGet here? we are returning an error to be thrown later, in SubmitPowerReport

Reply:
I see, saw why we are returning an error below. In this case, can you also change `_getInactivePower(rt Runtime)` to `_getInactivePower()` in this same PR?

Reply:
Done.